bin/setup_flat

```diff
@@ -72,7 +86,11 @@ container=forch-faucet-$inst
 container_inst=inst/$container/
 mkdir -p $container_inst/
 cp -r topo/$topo/faucet $container_inst/
+setup_flat_faucet_config $switches
+cp topo/common/faucet/ryu.conf $container_inst/faucet/
```

Comment:
common copy should go before the template additions.

Reply:
Done